src/errors/file-protocol-prefix.ts

```diff
@@ -0,0 +1,3 @@
+const FILE_PROTOCOL_PREFIX = 'file:///';
```

Comment:
Move to the shared place and refactor [this](https://github.com/DevExpress/testcafe/blob/master/src/client/browser/index.js#L18) code too.